languages.yaml

```diff
@@ -1,10 +1,29 @@
+de:
+  PLUGIN_TRANSLATE_DATE:
+    F: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember']
+    M: ['Jan', 'Feb', 'Mär', 'Apr', 'Mai', 'Jun', 'Jul', 'Aug', 'Sep', 'Okt', 'Nov', 'Dez']
+    l: ['Montag', 'Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag', 'Sonntag']
+    D: ['Mon', 'Die', 'Mit', 'Don', 'Fre', 'Sam', 'Son']
+
+    PROCESSOR: Prozessor
```

Comment:
I would prefer to not mix up formats translations with other non-format strings. Also nested structure is more readable and future proof

Reply:
The convention is to have the plugin prefix followed by the translation element, e.g. PLUGIN_TRANSLATE_DATE.PROCESSOR. So if you don't like it, it would make sense to encapsulate the format translations imo.

Reply:
Did you think about it? What would be your favorite way?

Reply:
It's on my todo list. I would still prefer the way I explained earlier to make it easier for myself to maintain. I don't want to merge it the way it is currently and I didn't find the time to get to it yet 🙁

Reply:
What about going back one step and leaving the admin interface untranslated? That would also be okay for me and many other plugins also come only with an English admin interface.

Reply:
I guess it would work for now. I might get to those some day though